playbooks/roles/openstack-remove-compute/tasks/main.yml

```diff
@@ -14,6 +14,10 @@
   delegate_to: localhost
   when: compute_id.stdout | length | int == 0
 
+- debug:
```

Comment:
should this warning to be added to the run.sh and require user confirmation instead?

Reply:
Fixed